src/app/components/LiveLabel/index.tsx

```diff
@@ -7,23 +7,44 @@ import { ServiceContext } from '../../contexts/ServiceContext';
 import styles from './index.styles';
 
 interface LiveLabelProps {
-  ariaHidden?: boolean;
-  liveText?: string;
   offScreenText?: string;
   lang?: string;
   id?: string;
 }
 
 const LiveLabel = ({
-  ariaHidden = false,
-  liveText = 'LIVE',
-  offScreenText = '',
   lang = 'en-GB',
   id,
   children,
+  offScreenText,
 }: PropsWithChildren<LiveLabelProps>) => {
-  const { dir } = useContext(ServiceContext);
+  const { dir, translations } = useContext(ServiceContext);
 
+  const { liveLabel } = translations.media;
+
+  // As screenreaders mispronounce the word 'LIVE', we use visually hidden
+  // text to read 'Live' instead, which screenreaders pronounce correctly.
+  const liveLabelIsEnglish = liveLabel === 'LIVE';
+  let screenReaderText;
+  let ariaHidden = false;
+
+  // If the translated phrase for "LIVE" is English, we don't want this to be read out by the screenreader, as it is often pronounced incorrectly (rhyming with "give" instead of "hive"), hence why we set ariaHidden to true
+  if (liveLabelIsEnglish) {
+    ariaHidden = true;
+  }
+  // If offscreenText has been provided as a prop to the LiveLabel component then we want to include a pause after the offscreen text (hence the addition of a comma and a space to the text)
+  if (offScreenText) {
+    screenReaderText = `${offScreenText}, `;
+    console.log('i am lily lol');
+    // ariaHidden = true;
```

Comment:
As we have seen when testing the radio schedule screenreader UX, I think we need to also need to set the ariaHidden = true if `offscreen` text is provided - because we don't want both to be read out by the screenreader...

Did this not work?
```suggestion
    ariaHidden = true;
```

Reply:
this now works, we were using voiceover in firefox which has strange behaviour so didn't think it worked, but it does in safari which is the recommended browser for voiceover